app/models/project.rb

```diff
@@ -95,29 +158,17 @@ def self.approved_projects
   def self.data_user_projects(user)
     # See https://scalegrid.io/blog/using-jsonb-in-postgresql-how-to-effectively-store-index-json-data-in-postgresql/
     # for information on the @> operator
-    query_ro = '{"data_user_read_only":["' + user + '"]}'
-    query_rw = '{"data_user_read_write":["' + user + '"]}'
+    query_ro = '{"ro_users":["' + user + '"]}'
```

Comment:
This is changing what is being store in the database.  This means you need a migration for all existing projects, or we need to clear all projects.  What made you decide to store the information in the database differently?

Reply:
Restored them.